syntaxes/test/data/template.html.snap

```diff
@@ -149,6 +149,8 @@
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng
 ><div [invalidProperty)="val"></div>
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng
+><div>[my-property]="val"</div>
```

Comment:
I don't think we should have an incomplete solution to the problem. If this isn't working for multiline, then we should come up with a different way to address it. I expect the most common case will be a binding on a different line anyways. There's probably some way to make it work by only applying the property binding when in a given scope.

Closing this PR since it's not quite what we really need.